services/QuillLMS/app/controllers/api/v1/activity_sessions_controller.rb

```diff
@@ -97,6 +99,13 @@ def destroy
     @activity_session = ActivitySession.unscoped.find_by_uid!(params[:id])
   end
 
+  private def count_completed_activity
+    return unless @activity_session.finished?
+
+    counter = UserActivityClassificationCount.find_or_create_by(user: @activity_session.user, activity_classification: @activity_session.classification)
```

Comment:
`UserActivityClassification` typo?

Reply:
This typo reminded me that I was actually thinking about suggesting renaming the table `UserActivityClassificationCounter` which would express more clearly what the model is doing while also reserving the standard join table name `UserActivityClassification` for a more generic usage down the road.